src/Administration/Resources/app/administration/src/module/sw-category/component/sw-category-layout-card/index.js

```diff
@@ -92,5 +93,29 @@ export default {
         closeLayoutModal() {
             this.showLayoutSelectionModal = false;
         },
+
+        hasCmsPageOverrides() {
```

Comment:
Looks like this check is always true for certain elements, like the `sw-text-editor`